doc/user-docs/index.md

```diff
@@ -18,12 +18,10 @@ To compile Windows Terminal yourself using the source code, follow the instructi
 
 ## Starting Windows Terminal
 
-From the Windows Start menu, select Windows Terminal and run the application.
-
-Note: You can right click on the application item and run with Windows Administrator privilege if required.
-
-The default shell is PowerShell.
+1. Locate the _Windows Terminal_ app in your Start menu.
+2. Click _Windows Terminal_ to launch the app. If you need administrative privileges, right-click the entry and click `Run as administrator`.
```

Comment:
Don't know if its worth calling this out, but one can also launch Terminal elevated by also holding <kbd>CTRL</kbd>+<kbd>SHIFT</kbd> while launching the app

Reply:
Yep, that's pretty handy. Added a note about that.